wd_helper.au3

```diff
@@ -696,14 +696,34 @@ EndFunc   ;==>_WD_HighlightElement
 ; Example .......: No
 ; ===============================================================================================================================
 Func _WD_HighlightElements($sSession, $aElements, $iMethod = Default)
+	Local Const $sFuncName = "_WD_HighlightElements"
+	Local Const $aMethod[] = _
```

Comment:
The same `$aMethod[]` is used in `_WD_HighlightElement()` maybe should we put this to Globoal  scope ?
Or even can we combine this 2 separate function in one and only check if `IsString($vElement)` or `IsArray($vElement)`  ?
All others looks fine and should work properly.

Reply:
I was thinking the same thing about combining the functions.

Reply:
Hi @mlipok ,

my tendece is to agree about merging the functions together, because the differences aren't much.
This isn't usually my approach, but at this point could be okay 🤞 .

Best regards
Sven

Reply:
Check out the latest commit where I've essentially combined the functionality into a single function. `_WD_HighlightElement` still exists, but it just passes the parameters on to `_WD_HighlightElements`.

Under this scenario, `_WD_HighlightElement` could be marked as deprecated and it would eventually be removed.